src/prototype_room_creepbuilder.js

```diff
@@ -54,6 +54,8 @@ Room.prototype.spawnCheckForCreate = function() {
   creep.ttl = creep.ttl || config.creep.queueTtl;
   if (this.findSpawnsNotSpawning().length === 0) {
     creep.ttl--;
+  } else if (this.energyAvailable === this.energyCapacityAvailable) {
```

Comment:
please look into  `.checkRoleToSpawn('claimer'`

Reply:
Nah, this would clear the spawning queue, when all extensions are filled and all spawns are currently spawning.